benchadapt/python/benchadapt/result.py

```diff
@@ -63,18 +63,45 @@ class BenchmarkResult:
         For benchmarks run on a cluster, information about the cluster
     context : Dict[str, Any]
         Should include ``benchmark_language`` and other relevant metadata like compiler flags
-    github : Dict[str, Any]
-        Keys: ``repository`` (in the format ``org/repo``), ``commit``, and ``pr_number``.
-        If this is a benchmark on the default branch, you may leave out ``pr_number``.
-        If it's a non-default-branch & non-PR commit, you may supply the branch name to
-        the optional ``branch`` key in the format ``org:branch``.
+    github : Union[Optional[Dict[str, Any]], Literal["inspect_git_in_cwd"]
```

Comment:
new type annotation looks funny.

Reply:
Instead of a literal, could we just have users call the util function that scrapes and returns the dict?

Reply:
I am just removing this now. We don't use it anyway. I want to get over with this :).